mycity/platforms/amazon/models/en_US.json

```diff
@@ -1002,9 +1002,16 @@
                                 "It's {Address}",
                                 "{Address}"
                             ]
+                        },
+                        {
+                            "name": "number_incidents",
+                            "type": "AMAZON.NUMBER"
                         }
                     ],
                     "samples": [
+                        "most recent {number_incidents} crime activity",
+                        "top {number_incidents} incidents happened near me",
+                        "top {number_incidents} crime activity",
```

Comment:
Let's add some more options here. This are things I tried when testing the skill before looking at this list, but try to brainstorm as many as you can:
- what are the four most recent crime reports (goes to 311 requests intent)
- give me two crime reports (this gave me the default number of reports)
- what two crimes were reported (failed to find intent)

In addition, I don't think we shouldn't use "top" since that implies some sort of ranking. The only way we currently sort is by most recent reported date.